cmd/pipeline/deploy.go

```diff
@@ -34,8 +34,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// DeployOptions options for deploy pipeline command
-type DeployOptions struct {
+// flags represents the user input
+type flags struct {
```

Comment:
❤️

Reply:
I've also loved the way the codes organized. It looks much cleaner and helps to reduce dependencies with other packages